test/registry.ts

```diff
@@ -0,0 +1,131 @@
+import { ethers } from "hardhat";
+import { Contract, ContractFactory } from "@ethersproject/contracts";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+
+import {
+	generateFuncIdAsBytes,
+    getRandomNum
+} from './utils/utils';
+
+const REFUNDER_VERSION = 1;
+
+const randomNum = getRandomNum();
+
+describe('Registry', () => {
```

Comment:
We should add tests for checking that `updateRefundable` can be called only by refunder